QuizApp/LoginView/LoginViewController.swift

```diff
@@ -107,3 +95,20 @@ class LoginViewController: UIViewController {
     }
 
 }
+
+extension LoginViewController: LoginPresenterProtocol {
+    func loginSuccess() {
+        loginButton.isEnabled = true
+        print("Sucessful login")
+        errorLable.isHidden = true
+        router.showQuizzes()
```

Comment:
It's ok for the presenter to know about the router rather than the view controller.